runtime/etc/ntp.conf

```diff
@@ -0,0 +1,46 @@
+# /etc/ntp.conf, configuration for ntpd; see ntp.conf(5) for help
+
+driftfile /var/lib/ntp/ntp.drift
+
+
+# Enable this if you want statistics to be logged.
+#statsdir /var/log/ntpstats/
+
+statistics loopstats peerstats clockstats
+filegen loopstats file loopstats type day enable
+filegen peerstats file peerstats type day enable
+filegen clockstats file clockstats type day enable
+
+# Specify one or more NTP servers.
+# Use AWS Time Sync Service
+server 169.254.169.123
```

Comment:
@aermakov-zalando Despite using AWS Time Sync Service, we still observed drifting between instances freshly created, taking a long time to heal by themselves.

A freshly deploy instance:
```
remote           refid      st t when poll reach   delay   offset  jitter
==============================================================================
*******                      3 u   11   64   37    0.095  206.255 104.590
```

Same instance after 30 minutes:
```
remote           refid      st t when poll reach   delay   offset  jitter
==============================================================================
*******                      3 u   11   64  377    0.122   32.242  42.379
```

I have checked AWS documentation and suggests to use `prefer iburst` in order to force a fast synchronisation, why this option is not enabled?:
https://aws.amazon.com/blogs/aws/keeping-time-with-amazon-time-sync-service/

Reply:
Thanks for the suggestion, we'll enable `iburst` to ensure that the time syncs up more quickly after startup.

However, I would recommend against relying on the system clock being perfectly synchronised. AWS doesn't provide an SLA for the Time Sync Service or even any guarantees about the clock precision, you could run into issues with ntpd (which nobody actually monitors), and your application might still generate slightly incorrect timestamps due to GC pauses, network failures or other instance-related issues.

Reply:
PR: https://github.com/zalando-stups/taupage/pull/498

Reply:
@aermakov-zalando Kudos for the quick action!!

Unfortunately, we have to relay in the wall clock since my team is heavily invested in Cassandra and e.g: [Why Cassandra cluster need synchronized clocks between nodes?](https://stackoverflow.com/questions/34898693/why-cassandra-cluster-need-synchronized-clocks-between-nodes)

The time drifts causes to lose some writes and eventually has customer impact.